pom.xml

```diff
@@ -462,15 +462,15 @@
         <junit.version>4.12</junit.version>
         <metrics.version>3.1.2</metrics.version>
         <metrics.version.range>[3.1.0, 4.0.0)</metrics.version.range>
-        <eclipse.osgi.version>3.9.1.v20140110-1610</eclipse.osgi.version>
-        <equinox.osgi.services.version>3.3.100.v20130513-1956</equinox.osgi.services.version>
-        <carbon.kernel.version>4.4.7</carbon.kernel.version>
+        <eclipse.osgi.version>3.11.0.v20160603-1336</eclipse.osgi.version>
+        <equinox.osgi.services.version>3.5.100.v20160504-1419</equinox.osgi.services.version>
+        <carbon.kernel.version>4.4.37-SNAPSHOT</carbon.kernel.version> <!-- carbon-kernel branch: 4.4.x_java10 -->
```

Comment:
Why SNAPSHOT?

Reply:
They are WIP on a set of feature branches and will not be merged to main code path until all the libraries are updated. 
Where to find that dependency is specified as a comment in the pom itself.

Reply:
I will add it to the description